### PR TITLE
Manga Weebs: remove tag from genre

### LINKED ISSUE #8881: Manga Weebs: Remove tags from genre in app
### Describe your suggested feature

Genre & tags of the site are combined. PLS remove the tags so that it will look more cleaner in app. 
Screenshot_20210828-130247_Tachiyomi~2

### Other details

_No response_

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with the app itself, I should be opening an issue in the [app repository](https://github.com/tachiyomiorg/tachiyomi/issues/new/choose).
- [X] I have updated the app to version **[0.12.1](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [X] I will fill out all of the requested information in this form.

### PR DESCRIPTION
**Pull request by Riztard**
Closes #8881

Checklist:

- [x] Updated `extVersionCode` value in `build.gradle` for individual extensions
- [ ] Updated `overrideVersionCode` or `baseVersionCode` as needed for all multisrc extensions
- [x] Referenced all related issues in the PR body (e.g. "Closes #xyz")
- [ ] Added the `isNsfw = true` flag in `build.gradle` when appropriate
- [x] Have not changed source names
- [ ] Have explicitly kept the `id` if a source's name or language were changed
- [x] Have tested the modifications by compiling and running the extension through Android Studio

### CODE REVIEW

**Comment by AntsyLich** [comment]
Can you make it a conditional statement in madara theme to exclude tag from genre? Some other sources also uses tag for seo stuff. ReaperScans, HelaScan, TudoQuadrinhos and also DrakeScans which i was gonna work on. Should only need to bump version of MangaWeebs and drake scans in this case since this isn't a major theme change

**Comment by Riztard** [comment]
you mean make it true/false?
just like hasProjectPage ?

**Comment by alessandrojean** [comment]
You can just override the tag selector and made it a empty string, it's already handled by the Madara code IIRC. I did that for some Brazilian extensions before.

File: multisrc/overrides/madara/mangaweebs/src/MangaWeebs.kt [hunk 1]
```diff
@@ -5,5 +5,8 @@ import java.text.SimpleDateFormat
 import java.util.Locale
 
 class MangaWeebs : Madara("Manga Weebs", "https://mangaweebs.in", "en", dateFormat = SimpleDateFormat("dd MMMM HH:mm", Locale.US)) {
+
+    override val mangaDetailsSelectorTag = "not used"
```
[the remark this answers is not in the document]
Comment (Riztard) [reply multisrc/overrides/madara/mangaweebs/src/MangaWeebs.kt]: i thought it will be the same

**Pull request merged**